jupyter_releaser/actions/publish_release.py

```diff
@@ -8,9 +8,9 @@
 
 if release_url:
     run(f"jupyter-releaser extract-release {release_url}")
-    run(f"jupyter-releaser forwardport-changelog {release_url}")
 
 run(f"jupyter-releaser publish-assets {release_url}")
 
 if release_url:
     run(f"jupyter-releaser publish-release {release_url}")
+    run(f"jupyter-releaser forwardport-changelog {release_url}")
```

Comment:
Looks like the `release_url` (draft URL) will not be available anymore after running `publish-release` the line before?

Reply:
cc @martinRenou who noticed this in https://github.com/martinRenou/jupyter_releaser/runs/5508477170?check_suite_focus=true